pkg/roachprod/install/cluster_synced.go

```diff
@@ -1722,24 +1732,25 @@ func (c *SyncedCluster) createTenantCertBundle(
 				cmd += fmt.Sprintf(`cd %s ; `, c.localVMDir(1))
 			}
 			cmd += fmt.Sprintf(`
-CERT_DIR=tenant-certs/certs
-CA_KEY=%[1]s/ca.key
+CERT_DIR=%[1]s-%[5]d/certs
+CA_KEY=%[2]s/ca.key
 
 rm -fr $CERT_DIR
 mkdir -p $CERT_DIR
-cp %[1]s/ca.crt $CERT_DIR
+cp %[2]s/ca.crt $CERT_DIR
 SHARED_ARGS="--certs-dir=$CERT_DIR --ca-key=$CA_KEY"
-VERSION=$(%[2]s version --build-tag)
+VERSION=$(%[3]s version --build-tag)
 VERSION=${VERSION::3}
 TENANT_SCOPE_OPT=""
 if [[ $VERSION = v22 ]]; then
-        TENANT_SCOPE_OPT="--tenant-scope %[4]d"
+        TENANT_SCOPE_OPT="--tenant-scope %[5]d"
 fi
-%[2]s cert create-node %[3]s $SHARED_ARGS
-%[2]s cert create-tenant-client %[4]d %[3]s $SHARED_ARGS
-%[2]s cert create-client root $TENANT_SCOPE_OPT $SHARED_ARGS
-tar cvf %[5]s $CERT_DIR
+%[3]s cert create-node %[4]s $SHARED_ARGS
```

Comment:
This comment might be because of something I'm doing in my LB PR. But from what I can see, the node cert is created first here, and tends to always have the same arguments, which are the "nodeNames" (IPs & hostnames). I'm curious if we can share it, and have it in a common place. Mostly my ask since I need to update this cert when a cluster is growing, or a load balancer IP needs to be added to it.

Reply:
Just to make sure I'm following: are you talking about the duplicate `create-node` call in the functions that set up system vs app tenants?

We could clean this up, but might be better to do it in isolation. Unless I misunderstand what you are requesting, please let me know.

Reply:
Yes, I think it was present prior to this PR in any case. But all in all it would be nice to only have to update one node cert, if possible.

Reply:
Makes sense, can be done in another PR.